shielder/cli/src/ink_contract.rs

```diff
@@ -0,0 +1,282 @@
+use scale::Encode as _;
```

Comment:
This gets generated, correct? If so, maybe it's worth adding a header here (to be done in the `ink-wrapper` itself) noting the fact. We probably want to get it (the ink-wrapper) to a state where users of the generated code don't have to modify it. /cc @obrok

Reply:
You mean a general comment at the top of this file saying it's autogenerated?

Reply:
Yes

Reply:
Yeah, that's a good idea